### PR TITLE
ref(ui): Codeplit "ios-device-list"

### PR DESCRIPTION
**Pull request by billyvg**
Refactors components that use util func `deviceNameMapper` to use a new React Component
that dynamically imports `deviceNameMapper`. Displays original original device string while it loads `deviceNameMapper`.

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/components/deviceName.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  constructor(...args) {
+    super(...args);
+    this.state = {
+      deviceNameMapper: null,
+    };
+  }
+
+  componentWillMount() {
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/deviceName.jsx]: Maybe worth adding a comment that `ios-device-list` is a huge lib and should be loaded this way?

**Review by benvinegar — APPROVED** [review]
In practice, in real loading situations, do we see content popping/moving as a result of the text transforming as `ios-device-list` loads / triggers a re-render?

**Comment by billyvg** [comment]
I have not tested in a staging env, but it's definitely possible that content moves after `ios-device-list` is loaded. I think this is acceptable given 1) it only happens for ios devices and 2) only on first load

File: src/sentry/static/sentry/app/utils/deviceNameMapper.jsx [hunk 2]
```diff
@@ -1,15 +0,0 @@
-import iOSDeviceList from 'ios-device-list';
-import {isString} from 'lodash';
-
-export default function(model) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils/deviceNameMapper.jsx]: I'm removing this module, otherwise it's too easy to import normally and use.

**Pull request merged**